docs/Upgrade.md

```diff
@@ -437,6 +436,43 @@ const PostEdit = () => (
 );
 ```
 
+## DataProvider query functions are provided with a new `signal` parameter
```

Comment:
I prefer to keep the upgrade guide short, to avoid scaring off developers. We'll publish a post with the new features, and maybe invite each maintainer of a data provider package to implement the abort signal.

Reply:
Okay I've moved the instructions to this PR's description then, in case we need them for later. Should be all good now! :slightly_smiling_face: